src/ui/components/InventoryItem.module.scss

```diff
@@ -0,0 +1,41 @@
+.itemContainer {
+  width: 280px;
+  min-height: 120px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5px;
+  padding: 5px 10px 10px;
+  justify-self: center;
+  border-radius: 8px;
+}
+
+.itemContainer > h2 {
+  font-size: 18px;
+  text-transform: capitalize;
+}
+
+.itemContainer > p {
+  font-size: 14px;
+  text-transform: capitalize;
+}
```

Comment:
Evite targetar tags, sempre prefira classes, porque targetando tags a gente acopla o nosso CSS à tag específica que estamos usando o que quase nunca é o que queremos.

Por exemplo, se alguém for mexer nesse código e tiver que mudar o `p` para um `span` ou então o `h2` por um `h3`, o código vai quebrar, ao passo que se a gente usar classes somente, isto não acontece.

Existem casos de uso pra usar esse tipo de seletor, mas eles são bem específicos, como por exemplo a gente precisa targetar elementos de uma lib cujo código a gente não controla e, ainda assim, é bastante frágil e a gente só usa em último caso.